transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/FeeFormatter.java

```diff
@@ -48,10 +48,13 @@ private static String getSingleFormattedFee(@NonNull Context context, @NonNull L
             String formattedString) {
         Fee fee = fees.get(0);
         if (Fee.FeeRate.FLAT.equals(fee.getFeeRateType())) {
+            if (!TextUtils.isEmpty(fee.getValue()) && Double.parseDouble(fee.getValue()) == 0.0) {
```

Comment:
`if (!TextUtils.isEmpty(fee.getValue()) && Double.parseDouble(fee.getValue()) == 0.0)` is 0.0 or just 0?

Reply:
Double.parseDouble return as 0.0 not 0 (if 0,0.0,0.00,0.000)